stanford_news.info.yml

```diff
@@ -21,10 +19,14 @@ dependencies:
   - pathauto
   - rdf
   - scheduler
+  - stanford_news
```

Comment:
lol. i bet that was from rebuilding and it renamed the exported module to `Stanford_stanford_news`